src/app/views/marketplace/marketplace-app.html

```diff
@@ -264,7 +264,8 @@ <h2 class="media-heading">{{vm.app.name}}</h2>
                       </div>
                       <div class="col-md-10">
                         <p class="question-text">{{ question.description }}</p>
-                        <p class="question-attribute">Posted by: {{ question.user_name }} in {{ question.organization_name }} on {{ question.created_at | date: "dd/MM/yyyy" }}</p>
+                        <p ng-hide="question.user_admin_role == 'admin'" class="question-attribute">Posted by: {{ question.user_name }} in {{ question.organization_name }} on {{ question.created_at | date: "dd/MM/yyyy" }}</p>
```

Comment:
@alexnoox For consistency should we use `ng-if`/`ng-show` for both comment and questions? Which one is better?